### PR TITLE
fix: fix loader not centered horizontally

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Copy of https://github.com/containers/podman-desktop/pull/3269 for 1.2.x version

### Screenshot/screencast of this PR

image

### How to test this PR?

see instructions from https://github.com/containers/podman-desktop/pull/3269

Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

**Review by lstocchi — APPROVED** [review]

**Comment by benoitf** [comment]
@deboer-tim you wanted to change the content to get rid of patternfly

you want to do it as part of this PR or a follow-up PR ?

**Review by cdrage — APPROVED** [review]

**Comment by deboer-tim** [comment]
I would just do the following. Do you want to commit or should I do a separate PR? Not sure how to 'take over'.
```
  <main class="flex flex-row w-screen h-screen justify-center">
    <div class="flex flex-col justify-center">
      <SealRocket />
      <h1 class="text-center text-xl">Initializing...</h1>
    </div>
  </main>
```

**Comment by benoitf** [comment]
@deboer-tim you can just add new commits on top of my branch, PR will be updated 👍

File: packages/renderer/src/Loader.svelte [hunk 1]
```diff
@@ -82,14 +82,10 @@ window.events.receive('starting-extensions', (value: string) => {
...
 {#if !systemReady}
-  <main class="min-h-screen flex flex-col h-screen">
-    <div class="min-h-full min-w-full flex flex-col">
-      <div class="pf-c-empty-state h-full">
-        <div class="pf-c-empty-state__content">
-          <SealRocket />
-          <h1 class="pf-c-title pf-m-lg">Initializing...</h1>
-        </div>
-      </div>
+  <main class="flex flex-row w-screen h-screen justify-center">
```
Comment (benoitf) [inline packages/renderer/src/Loader.svelte]:
```suggestion
  <main class="flex flex-row w-screen h-screen justify-center" style="-webkit-app-region: drag;">
```

@deboer-tim what about adding the drag option so we can move the window ?

**Pull request merged**